src/renderer/views/Watch/Watch.js

```diff
@@ -1186,11 +1188,15 @@ export default defineComponent({
 
       this.playlistId = this.$route.query.playlistId
       this.playlistItemId = this.$route.query.playlistItemId
+      this.isInvidiousPlaylist = this.$route.query.playlistType === 'invidious'
+      this.origin = this.$route.query.origin
 
       if (this.playlistId == null || this.playlistId.length === 0) {
         this.playlistType = ''
         this.playlistItemId = null
         this.watchingPlaylist = false
+        this.playlistId = this.$route.query.playlistId
+        this.playlistItemId = this.$route.query.playlistItemId
```

Comment:
These two lines look identical to the lines, only a few lines further up. Are these needed?